src/main/java/com/linkmoa/domain/memberPageLink/entity/MemberPageLink.java

```diff
@@ -0,0 +1,64 @@
+package com.linkmoa.domain.memberPageLink.entity;
+
+import com.linkmoa.common.entity.BaseEntity;
+import com.linkmoa.domain.member.entity.Member;
+import com.linkmoa.domain.page.entity.Page;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@Table(name = "member_page_link")
```

Comment:
Nit: 위에 작성하신 `directories`로 보아서 DB 테이블명 컨벤션이 기본적으로 복수형을 따라가는 것 같은데, 단수형 혹은 복수형 하나로 통일해보면 좋을 것 같습니다 :thinking: 

```suggestion
@Table(name = "member_page_links")
```